File: C++/Human.cpp
#include <bits/stdc++.h>
#include "Mahasiswa.cpp"

using namespace std;

class Human : public Mahasiswa{
    
    private:
        string NIK;
        string nama;
        string jenis_kelamin;

    public:
        Human(){
            this->NIK = "";
            this->nama = "";
            this->jenis_kelamin = "";
        }

        void set_NIK(string NIK){
            this->NIK = NIK;
        }

        string get_NIK(){
            return this->NIK;

        }

        void set_nama(string nama){
            this->nama = nama;
        }

        string get_nama(){
            return this->nama;
        }
        
        void set_jenis_kelamin(string jenis_kelamin){
            this->jenis_kelamin = jenis_kelamin;
        }

        string get_jenis_kelamin(){
            return this->jenis_kelamin;
        }
};

File: Java/Human.java
public class Human
{
    private String NIK;
    private String nama;
    private String jenis_kelamin;

    public Human(){
        this.NIK = "";
        this.nama = "";
        this.jenis_kelamin = "";
    }

    public String get_NIK(){
        return this.NIK;
    }
    public void set_NIK(String NIK){
        this.NIK = NIK; 
    }

    public String get_nama(){
        return this.nama;
    }
    public void set_nama(String nama){
        this.nama = nama; 
    }

    public String get_jenis_kelamin(){
        return this.jenis_kelamin;
    }
    public void set_jenis_kelamin(String jenis_kelamin){
        this.jenis_kelamin = jenis_kelamin;
    }
}